run large row workload panic

### What did you do?

```
./workload -database-host 127.0.0.1 -database-port 4000 -database-db-name test -total-row-count 100000000 -table-count 1 -workload-type large_row -thread 64 -action write -percentage-for-update 0.2 -row-size 4096
```


### What did you expect to see?

Workload runs normally.

### What did you see instead?


```
panic: runtime error: index out of range [-2]

goroutine 95 [running]:
math/rand.(*rngSource).Uint64(...)
	/snap/go/11017/src/math/rand/rng.go:249
math/rand.(*rngSource).Int63(0x503442?)
	/snap/go/11017/src/math/rand/rng.go:234 +0x85
math/rand.(*Rand).Int63(...)
	/snap/go/11017/src/math/rand/rand.go:96
math/rand.(*Rand).Int(...)
	/snap/go/11017/src/math/rand/rand.go:114
workload/schema/largerow.(*LargeRowWorkload).getSmallRow(0xc0000c42d0)
	/home/ubuntu/ticdc/tools/workload/schema/largerow/large_row.go:69 +0x25
workload/schema/largerow.(*LargeRowWorkload).BuildUpdateSql(0xc0000c42d0, {0xc0000308c0?, 0xc00f5de000?, 0xe0?, 0xc00045c700?})
	/home/ubuntu/ticdc/tools/workload/schema/largerow/large_row.go:157 +0x4b7
main.(*WorkloadApp).executeRegularUpdate(0xc0000ae3f0, 0xc0000308c0, {{0x0, 0xa, 0x0, 0x5}, {0x0, 0x0}, 0x0})
	/home/ubuntu/ticdc/tools/workload/update.go:190 +0x5f
main.(*WorkloadApp).executeUpdate(0x0?, 0xc000464068?, {{0x0, 0xa, 0x0, 0x5}, {0x0, 0x0}, 0x0})
	/home/ubuntu/ticdc/tools/workload/update.go:168 +0xc5
main.(*WorkloadApp).processUpdateTask(0xc0000ae3f0, 0xc0000308c0?, {{0x0, 0xa, 0x0, 0x5}, {0x0, 0x0}, 0x0})
	/home/ubuntu/ticdc/tools/workload/update.go:142 +0x65
main.(*WorkloadApp).doUpdate(0xc0000ae3f0, 0xc0000308c0, 0xc000464000)
	/home/ubuntu/ticdc/tools/workload/update.go:129 +0x10b
main.(*WorkloadApp).executeUpdateWorkers.func2(0x5)
	/home/ubuntu/ticdc/tools/workload/update.go:84 +0x390
created by main.(*WorkloadApp).executeUpdateWorkers in goroutine 1
	/home/ubuntu/ticdc/tools/workload/update.go:64 +0x351
```

### Versions of the cluster

Upstream TiDB cluster version (execute `SELECT tidb_version();` in a MySQL client):

```console
(paste TiDB cluster version here)
```

Upstream TiKV version (execute `tikv-server --version`):

```console
(paste TiKV version here)
```

TiCDC version (execute `cdc version`):

```console
(paste TiCDC version here)
```

--- a/tools/workload/schema/largerow/large_row.go
+++ b/tools/workload/schema/largerow/large_row.go
@@ -21,11 +21,10 @@ import (
 	"sync/atomic"
 	"time"
 
-	"workload/schema"
-	"workload/util"
-
 	"github.com/pingcap/log"
 	"go.uber.org/zap"
+	"workload/schema"
+	"workload/util"
 )
 
 const varcharColumnMaxLen = 16383

